docker-compose.yaml

```diff
@@ -17,6 +17,17 @@ services:
       - 3902:3902
       - 3903:3903
 
+  witness:
+    container_name: idw-witness
+    restart: unless-stopped
+    build:
+      context: github.com/WebOfTrust/keripy#ae774b135585f850415f109c1fd0b294e823a2c9
+      dockerfile: ./images/witness.demo.dockerfile
```

Comment:
I'd rather not depend on the GLEIF image - can we use the normal keripy docker image from Dockerhub, with the start command `kli witness demo`

Similar to this: https://github.com/WebOfTrust/signify-ts/blob/0ad587316ba2d98093158508dfb1af38da796c31/docker-compose.yaml#L57

We can use `1.1.6` as this is the version matches our KERIA version right now

Reply:
@iFergal Yes. I have changed image